data/open-source-projects.json

```diff
@@ -59,7 +59,8 @@
         "PhantasWeng/coronavirus-daily-dashboard",
         "nytimes/covid-19-data",
         "cipriancraciun/covid19-datasets",
-        "pcm-dpc/COVID-19"
+        "pcm-dpc/COVID-19",
+        "colaboradados/colaboradados.github.io/blob/dev/_posts/2020-03-19-coronavirus.md"
```

Comment:
Hey @anapaulamendes, I believe here you only have to add your github repo so in the form of `colaboradados/colaboradados.github.io` and the README will be autogenerated to add the description in your github repo

Reply:
Done!